.github/workflows/build-push-docker.yaml

```diff
@@ -115,7 +111,7 @@ jobs:
           context: .
           file: "Dockerfile.${{ matrix.dockerfile }}"
           tags: ${{ steps.meta.outputs.tags }}
-          push: ${{ github.event_name != 'pull_request' }}
+          push: true
```

Comment:
This means we'll be building docker images on PRs as well, correct? To confirm, is this required/intentional?

Reply:
This is only a temporary change (alongside this temporary commit) so we can test the changes. I will revert this change after we've confirmed that the new image works as expected.

I just [opened this](https://github.com/nebari-dev/nebari-docker-images/issues/87) issue to discuss whether we want to push images to Quay on a PR trigger. Thanks for asking, this is a good point for us to discuss in more detail as a team.

Reply:
This change has now been reverted since the images are now available on Quay for us to test with :)